apps/qubit/modules/informationobject/actions/indexAction.class.php

```diff
@@ -42,7 +42,7 @@ public function execute($request)
 
         $this->dispatcher->notify(new sfEvent($this, 'access_log.view', ['object' => $this->resource]));
 
-        if ('fullWidth' == sfConfig::get('app_treeview_type__source', 'sidebar')) {
+        if (!sfConfig::get('app_b5_theme', false) && 'fullWidth' == sfConfig::get('app_treeview_type__source', 'sidebar')) {
```

Comment:
And how are they loaded from the hierarchy page?

Reply:
They are loaded using view.yml in the browse module.